EGaDS! Open Project Game/Assets/Scripts/EmployeeController.cs

```diff
@@ -0,0 +1,50 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EmployeeController : MonoBehaviour {
+
+    [SerializeField] private Transform[] waypoints;
+    // controls the movespeed of the employee
+    [SerializeField] private float moveSpeed = 10;
+    // true if the employee loops waypoints, false if it stops at the last waypoint 
+    [SerializeField] private bool looping = true;
+    // controls the amount of damage dealt to the player upon touching the employee
+    [SerializeField] private int damage = 1;
+    private int nextWaypoint = 1;
+    public Rigidbody2D rb;
+
+    // Start is called before the first frame update
+    void Start() {
+        // Sets the tired employee enemy to begin at the first waypoint
+        transform.position = waypoints[0].transform.position;
+    }
+
+    // Update is called once per frame
+    void Update() {
+        checkWaypoint();
+        transform.position = Vector2.MoveTowards(transform.position, waypoints[nextWaypoint].position, moveSpeed * Time.deltaTime);
+    }
+
+    // Checks if Employee has reached nextWaypoint and updates nextWaypoint
+    void checkWaypoint() {

```

Comment:
The naming convention for methods are `MethodName()`, so this should be `CheckWaypoint()`